docs/Backups.md

```diff
@@ -259,11 +259,11 @@ To backup the shared configuration:
 
 To restore a previous backup, copy the six files listed above to `/etc/clearwater` on one of your sprout nodes. Then run the following commands on that node:
 
```

Comment:
These look terrible - add sudo to the remaining three even though they don't need it just to stop it looking so inconsistent?